### ISSUE TITLE
No toolbar appears

### ISSUE THREAD

**Issue by afeldstein** [dsifford/academic-bloggers-toolkit#338]
**ABT Version:** 4.10.0

**PHP Version:** 5.6

**Theme: Fixed**

**Plugin List: all deactivated**
- Plugin 1
- Plugin 2

**Browser:** Firefox (Version: 51.0.1)

**Expected behavior:reference toolbar would appear when creating new post**

**Actual behavior:Box in top right when creating new post labelled 'reference list' under header a circular prompt that keeps changing color but no toolbar appears**

**Console messages:**
```js
// Paste messages here
JQMIGRATE: Migrate is installed, version 1.4.1  load-scripts.php:9:542
"Download the React DevTools for a better development experience: https://fb.me/react-devtools"  vendor.bundle.js:73:9

```

**Comment by dsifford** [comment]
@afeldstein What other plugins do you have installed?

**Comment by afeldstein** [comment]
Academic Blogger's Toolkit
Deactivate
<http://netorio.us/wp-admin/plugins.php?action=deactivate&plugin=academic-bloggers-toolkit%2Facademic-bloggers-toolkit.php&plugin_status=all&paged=1&s&_wpnonce=3a9332a428>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=academic-bloggers-toolkit/academic-bloggers-toolkit.php>
 | Plugin Settings
<http://netorio.us/wp-admin/options-general.php?page=abt-options>

A plugin extending the functionality of Wordpress for academic blogging
Version 4.10.0 | By Derek P Sifford <https://github.com/dsifford> | View
details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=academic-bloggers-toolkit&TB_iframe=true&width=600&height=550>
|
♥ Donate <https://donorbox.org/academic-bloggers-toolkit>
Select Akismet Anti-Spam Akismet Anti-Spam
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=akismet%2Fakismet.php&plugin_status=all&paged=1&s&_wpnonce=e4978a69f4>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=akismet/akismet.php> |
Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=akismet%2Fakismet.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

Used by millions, Akismet is quite possibly the best way in the world
to protect
your blog from spam. It keeps your site protected even while you sleep. To
get started: activate the Akismet plugin and then go to your Akismet
Settings page to set up your API key.
Version 3.3.4 | By Automattic
<https://automattic.com/wordpress-plugins/> | View
details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=akismet&TB_iframe=true&width=600&height=550>
Select Amazon Link Amazon Link
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=amazon-link%2Famazon.php&plugin_status=all&paged=1&s&_wpnonce=09a76be313>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=amazon-link/amazon.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=amazon-link%2Famazon.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

A plugin that provides a facility to insert Amazon product links directly
into your site's Pages, Posts, Widgets and Templates.
Version 3.2.7 | By Paul Stuttard <http://www.houseindorset.co.uk/> | View
details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=amazon-link&TB_iframe=true&width=600&height=550>
Select Dropbox Backup & Restore Dropbox Backup & Restore
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=dropbox-backup%2Fdropbox-backup.php&plugin_status=all&paged=1&s&_wpnonce=034e4a14ae>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=dropbox-backup/dropbox-backup.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=dropbox-backup%2Fdropbox-backup.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

Dropbox Backup & Restore Plugin to create Dropbox Full Backup (Files +
Database) of your Web Page
Version 1.5.1.4 | View details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=dropbox-backup&TB_iframe=true&width=600&height=550>


Select Jetpack by WordPress.com Jetpack by WordPress.com
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=jetpack%2Fjetpack.php&plugin_status=all&paged=1&s&_wpnonce=b42b632dfb>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=jetpack/jetpack.php> |
Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=jetpack%2Fjetpack.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

Bring the power of the WordPress.com cloud to your self-hosted WordPress.
Jetpack enables you to connect your blog to a WordPress.com account to use
the powerful features normally only available to WordPress.com users.
Version 5.2.1 | By Automattic <https://jetpack.com/> | View details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=jetpack&TB_iframe=true&width=600&height=550>
Select Limit Login Attempts Limit Login Attempts
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=limit-login-attempts%2Flimit-login-attempts.php&plugin_status=all&paged=1&s&_wpnonce=2bf44933d7>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=limit-login-attempts/limit-login-attempts.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=limit-login-attempts%2Flimit-login-attempts.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

Limit rate of login attempts, including by way of cookies, for each IP.
Version 1.7.1 | By Johan Eenfeldt <http://devel.kostdoktorn.se/> | View
details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=limit-login-attempts&TB_iframe=true&width=600&height=550>
Select Mendeley Plugin Mendeley Plugin
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=mendeleyplugin%2Fwp-mendeley.php&plugin_status=all&paged=1&s&_wpnonce=d90e3a6e50>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=mendeleyplugin/wp-mendeley.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=mendeleyplugin%2Fwp-mendeley.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

This plugin offers the possibility to load lists of document references
from Mendeley (shared) collections, and display them in WordPress posts or
pages.
Version 1.1.17 | By Michael Koch
<http://www.kooperationssysteme.de/personen/koch/> | View details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=mendeleyplugin&TB_iframe=true&width=600&height=550>

Select Pinterest WordPress Plugin - AccessPress Pinterest Pinterest
WordPress Plugin - AccessPress Pinterest
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=accesspress-pinterest%2Faccesspress-pinterest.php&plugin_status=all&paged=1&s&_wpnonce=f9cf5cc8e5>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=accesspress-pinterest/accesspress-pinterest.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=accesspress-pinterest%2Faccesspress-pinterest.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

A plugin to add various pinterest widgets and pins to a site with dynamic
configuration options.
Version 3.1.9 | By AccessPress Themes <http://accesspressthemes.com/> | View
details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=accesspress-pinterest&TB_iframe=true&width=600&height=550>
Select Recent Posts Widget With Thumbnails Recent Posts Widget With
Thumbnails
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=recent-posts-widget-with-thumbnails%2Frecent-posts-widget-with-thumbnails.php&plugin_status=all&paged=1&s&_wpnonce=54b047fac5>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=recent-posts-widget-with-thumbnails/recent-posts-widget-with-thumbnails.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=recent-posts-widget-with-thumbnails%2Frecent-posts-widget-with-thumbnails.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

Small and fast plugin to display in the sidebar a list of linked titles and
thumbnails of the most recent postings
Version 5.1 | By Martin Stehle <http://stehle-internet.de/> | View details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=recent-posts-widget-with-thumbnails&TB_iframe=true&width=600&height=550>
Select Under Construction Under Construction
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=underconstruction%2FunderConstruction.php&plugin_status=all&paged=1&s&_wpnonce=67db33a341>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=underconstruction/underConstruction.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=underconstruction%2FunderConstruction.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

Makes it so your site can only be accessed by users who log in. Useful for
developing a site on a live server, without the world being able to see it
Version 1.18 | By Noah Kagan <http://sumome.com/> | View details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=underconstruction&TB_iframe=true&width=600&height=550>
Select WordPress Importer WordPress Importer
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=wordpress-importer%2Fwordpress-importer.php&plugin_status=all&paged=1&s&_wpnonce=c2ac46c1b5>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=wordpress-importer/wordpress-importer.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=wordpress-importer%2Fwordpress-importer.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

Import posts, pages, comments, custom fields, categories, tags and more
from a WordPress export file.
Version 0.6.3 | By wordpressdotorg <http://wordpress.org/> | View details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=wordpress-importer&TB_iframe=true&width=600&height=550>
Select WP BackItUp Community Edition WP BackItUp Community Edition
Activate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=wp-backitup%2Fwp-backitup.php&plugin_status=all&paged=1&s&_wpnonce=008a5a34dc>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=wp-backitup/wp-backitup.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=wp-backitup%2Fwp-backitup.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>

Backup your content, settings, themes, plugins and media in just a few
simple clicks.
Version 1.14.7 | By WPBackItUp <https://www.wpbackitup.com/> | View details
<http://netorio.us/wp-admin/plugin-install.php?tab=plugin-information&plugin=wp-backitup&TB_iframe=true&width=600&height=550>
Select WPForms Lite WPForms Lite
SAActivate
<http://netorio.us/wp-admin/plugins.php?action=activate&plugin=wpforms-lite%2Fwpforms.php&plugin_status=all&paged=1&s&_wpnonce=60c3336e65>
 | Edit
<http://netorio.us/wp-admin/plugin-editor.php?file=wpforms-lite/wpforms.php>
 | Delete
<http://netorio.us/wp-admin/plugins.php?action=delete-selected&checked%5B0%5D=wpforms-lite%2Fwpforms.php&plugin_status=all&paged=1&s&_wpnonce=eecb2e4fc9>
All are currently deactivated.

Andrew Feldstein
Associate VP Learning Technologies Group
Fort Hays State University
Tel: 785 628-4788
Email: Andrew.Feldstein@fhsu.edu

**Comment by dsifford** [comment]
None of those plugins jump out at me, but if I had to guess, it's likely a conflict with either `WPForms Lite` or your theme. 

Can you deactivate all your plugins and let me know if the issue still persists?

Also, what theme do you have?

Thanks

**Comment by afeldstein** [comment]
I am using a theme called “Fixed” from Array.  I deactivated all plugins this morning when I first reported the issue. The issue persisted until about 10 minutes ago when, as I was working on a post, the plugin just started working.  I’ll activate plugins one by one to see if one of them is causing the issue.  Thank you for your quick response!

Any chance you’re working on a sidebar widget?

Andrew
